observatory_platform/utils/gc_utils.py

```diff
@@ -1,4 +1,5 @@
 # Copyright 2020 Curtin University
+# Copyright 2020 Artificial Dimensions Ltd
```

Comment:
I wrote `render_sql_query` for a project of my own and decided it would be useful for this project, so I just released it under the Apache license (hence the other copyright notice), I could perhaps move it (and other stuff related to jinja2) to it's own file so that it is clearer. It should probably be moved out of gc_utils.py anyway, especially if we make it more general.